tests/specs/mcd/verification.k

```diff
@@ -326,13 +350,18 @@ module LEMMAS-MCD-COMMON
 
     rule M:Memory [ K1 := (BUF11 ++ BUF12) ] [ K2 := BUF2 ] => M [ K1 +Int #sizeByteArray(BUF11) := BUF12 ] [ K2 := BUF2 ]
       requires K2 <=Int K1
-       andBool K1 +Int #sizeByteArray(BUF11) <=Int K2 +Int #sizeByteArray(BUF2)
+      andBool K1 +Int #sizeByteArray(BUF11) <=Int K2 +Int #sizeByteArray(BUF2)
       [simplification]
 
   // ########################
   // Arithmetic
   // ########################
 
+    rule X <=Int maxUInt48 &Int X +Int Y => false
```

Comment:
Add parens to make this clearer/more-explicit, so people don't have to think about precedence reading it.

Reply:
Yes.